components/scream/docs/build.md

```diff
@@ -0,0 +1,93 @@
+# Building and Testing Standalone SCREAM
+
```

Comment:
Thanks for moving this stuff from README.md to build.md. Makes much more sense. I think "standalone SCREAM" isn't quite the right word since this description allows us to test SCREAM unit tests...

Reply:
Fixed.